packages/uni_app/lib/view/academic_path/academic_path.dart

```diff
@@ -29,7 +30,10 @@ class AcademicPathPageViewState extends GeneralPageViewState {
   @override
   Widget getBody(BuildContext context) {
     return ListView(
-      children: academicPathCards,
+      children: [
+        const ConnectivityWarning(),
```

Comment:
To make the position of the ConnectivityWarning `fixed`, move it to the `getTopNavBar` method in `GeneralPageViewState` (however, note that the layout of the pages may change during the redesign).

Reply:
Following the conversation we had, I tried to change the `_createTopWodgets` method in the `top_navigation_bar.dart` class but there are 2 problems. The first is that the home page uses a getHeader method that creates the tittle differently from the other pages (the warning appears above instead of below the tittle). The second problem is that the size of the AppBar isnt dynamically changed when the warning is displayed, so it seems cut out. Do I have to increase its size or its my implementation that isnt correct?
<img src=https://github.com/user-attachments/assets/d3a4e8bf-d841-4b6f-9760-92d35a5c79ed width=250>
<img src=https://github.com/user-attachments/assets/174eb31d-2d68-41de-92e1-e6ffeb2fee87 width=250>